src/main/java/core/basesyntax/FigCol.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class FigCol implements Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the start of the class definition.